src/pool/srv_layout.c

```diff
@@ -43,72 +43,104 @@ RDB_STRING_KEY(ds_pool_prop_, scrub_freq);
 RDB_STRING_KEY(ds_pool_prop_, scrub_thresh);
 RDB_STRING_KEY(ds_pool_prop_, svc_redun_fac);
 RDB_STRING_KEY(ds_pool_prop_, obj_version);
+RDB_STRING_KEY(ds_pool_prop_, checkpoint_mode);
+RDB_STRING_KEY(ds_pool_prop_, checkpoint_freq);
+RDB_STRING_KEY(ds_pool_prop_, checkpoint_thresh);
 
 /** default properties, should cover all optional pool properties */
 struct daos_prop_entry pool_prop_entries_default[DAOS_PROP_PO_NUM] = {
-	{
-		.dpe_type	= DAOS_PROP_PO_LABEL,
-		.dpe_str	= DAOS_PROP_PO_LABEL_DEFAULT,
-	}, {
-		.dpe_type	= DAOS_PROP_PO_SPACE_RB,
-		.dpe_val	= 0,
-	}, {
-		.dpe_type	= DAOS_PROP_PO_SELF_HEAL,
-		.dpe_val	= DAOS_SELF_HEAL_AUTO_EXCLUDE |
-				  DAOS_SELF_HEAL_AUTO_REBUILD,
-	}, {
-		.dpe_type	= DAOS_PROP_PO_RECLAIM,
-		.dpe_val	= DAOS_RECLAIM_LAZY,
-	}, {
-		.dpe_type	= DAOS_PROP_PO_ACL,
-		.dpe_val_ptr	= NULL, /* generated dynamically */
-	}, {
-		.dpe_type	= DAOS_PROP_PO_OWNER,
-		.dpe_str	= "NOBODY@",
-	}, {
-		.dpe_type	= DAOS_PROP_PO_OWNER_GROUP,
-		.dpe_str	= "NOBODY@",
-	}, {
-		.dpe_type	= DAOS_PROP_PO_SVC_LIST,
-		.dpe_val_ptr	= NULL,
-	}, {
-		.dpe_type	= DAOS_PROP_PO_EC_CELL_SZ,
-		.dpe_val	= DAOS_EC_CELL_DEF,
-	}, {
-		.dpe_type	= DAOS_PROP_PO_REDUN_FAC,
-		.dpe_val	= DAOS_PROP_PO_REDUN_FAC_DEFAULT,
-	}, {
-		.dpe_type	= DAOS_PROP_PO_EC_PDA,
-		.dpe_val	= DAOS_PROP_PO_EC_PDA_DEFAULT,
+    {
```

Comment:
Not sure what happened here... Tabs vs. spaces thing?

Reply:
I am using clang-format with our settings in .clang-format so it modifies stuff you touch.  I can probably revert if preferred but I think we are moving in this direction and this is a reasonable change.

Reply:
OK. I was just wondering why so much had changed here. Might be worth having a flag day and just running the formatter over everything in one commit? In any case, I'm a big proponent of auto-formatting.